GPU/Vulkan/TextureCacheVulkan.cpp

```diff
@@ -84,15 +80,16 @@ float reduce(vec4 color) {
 }
 
 float DistYCbCr(vec4 pixA, vec4 pixB) {
-	const vec3 w = vec3(0.2627, 0.6780, 0.0593);
-	const float scaleB = 0.5 / (1.0 - w.b);
-	const float scaleR = 0.5 / (1.0 - w.r);
+	// https://en.wikipedia.org/wiki/YCbCr#ITU-R_BT.2020_conversion
+	const vec3 K = vec3(0.2627, 0.6780, 0.0593);
+	const mat3 MATRIX = mat3(K, 
+							-.5 * K.r / (1.0 - K.b),	-.5 * K.g / (1.0 - K.b),	.5,
```

Comment:
nit: I always recommend spaces for alignment, tabs for indent.  Tabs for alignment is where things get messy.

-[Unknown]

Reply:
I do too, but the rest of this raw string of formatted with tabs. Do you want me to replace all of them with spaces?

Reply:
Ah, I guess because it came from DeSmuME then.  Okay, then.

-[Unknown]